windows/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 list(APPEND HIDAPI_PUBLIC_HEADERS "hidapi_winapi.h")
+set(CMAKE_DEBUG_POSTFIX d)
```

Comment:
this is a huge change from CMake/build perspective
HIDAPI choosed not to use it since the beginning

Reply:
This is a great pity. With `CMAKE_DEBUG_POSTFIX` it is easier to distinguish debug and release builds. Otherwise, when using `INSTALL`, the two configurations overwrite each other. If you use CMake for further projects this option is not an API change either, because the exported targets are adapted as well.

Reply:
if that is required for your specific case - you can always set that variable as an additional CMake parameter:
`-DCMAKE_DEBUG_POSTFIX=d`
or, in case if you're using HIDAPI as a subfolder - set the veriable before adding HIDAPI as a subfolder

Reply:
Currently, I also solve it this way. Currently, I also solve it this way. I build HIDAPI separately and copy static and dynamic versions together into an SDK for further use.